privet/windows.go

```diff
@@ -9,25 +9,104 @@
 package privet
 
 import (
-	"errors"
+  "fmt"
+  "sync"
+  "github.com/andrewtj/dnssd"
```

Comment:
Does it make sense to let the Windows implementation bake in first before switching Mac also?